### DISCUSSION TITLE
Support for Roon Essentials

### DISCUSSION THREAD

**Discussion by kokobadu** [music-assistant/support#1120] [Feature requests and ideas]
Since users of Roon or Roon Essentials will not (I think) give up options that this software provides, it would be very handy to be able to mix Roon Essentials as music source with other sources available already via MA.

Thanks up front for considering this.

**Comment by kokobadu** [comment]
It could get label 'player support'.

**Reply by OzGav** [reply]
Actually it would be a music provider. Thanks for the suggestion. Will see how many others like this…

**Reply by livebelive** [reply]
count me in！

**Comment by silverbladeg** [comment]
+1

**Comment by milkplus27** [comment]
I'm a Roon user and would love to see this functionality in MA! +1

**Comment by marcelveldt** [comment]
I REALLY wonder why you would want to use both Roon and Music Assistant ?
Don't you just want to stick with one or the other ?

**Reply by milkplus27** [reply]
Pretty simple really - I like that MA integrates seamlessly with HA and I really enjoy the music discovery or Roon. If I could combine the two I would be happy.

**Reply by OzGav** [reply]
So what does Roon do with search that MA doesn’t?

**Reply by milkplus27** [reply]
It's not the search functionality that's the issue. Roon has a recommendation feature based on your listening habits that has introduced me to so much new music I never would have otherwise found. I link Qobuz to Roon and find it's much better than Qobuz's discovery algorithm.

If I could get Roon's recommendations in MA it would be perfect. But I realise I'm probably an edge case here...

**Reply by OzGav** [reply]
I assume you mean this feature https://valence.roonlabs.com/info

Seems quite advanced and therefore would be some way off for MA but never say never when Marcel is involved...!

**Comment by michelsup** [comment]
I wish I could switch from Roon to MA just to be able to use multiple streaming accounts (yes, in 2024 Roon doesn't support family accounts), but the lack of proper local library recommendations/Qobuz recommendations/continuous smart queue is a stopper.

**Reply by OzGav** [reply]
I understand Roon has a very good smart queue system that will take quite a while to replicate. However, MA does use streaming providers radio modes to implement the “Don’t Stop The Music” feature. Other recommendations are in the roadmap
